rotkehlchen/types.py

```diff
@@ -85,7 +85,7 @@
 AERODROME_POOL_PROTOCOL = 'aerodrome_pool'
 PICKLE_JAR_PROTOCOL = 'pickle_jar'
 SPAM_PROTOCOL = 'spam'
-GEARBOX_POOL_PROTOCOL = 'gearbox_pool'
+GEARBOX_POOL_PROTOCOL = 'gearbox'
```

Comment:
why this change? Doesn't it affect the assets in the asset repo?

Reply:
So this wasn't used anywhere in the code before, but the assets in the repo have been saved as protocol being `gearbox`. So, the change.